### PR TITLE
chart updated

### PR DESCRIPTION
**Pull request by SujithKasireddy**

### CODE REVIEW

**Comment by SujithKasireddy** [comment]
> _ No description provided. _

@nyrahul please review and merge it

File: knox-auto-policy-chart/configmapfiles/conf.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  level: "INFO"
+
+# kubectl -n kube-system port-forward service/hubble-relay --address 0.0.0.0 --address :: 4245:80
+cilium-hubble:
+  url: hubble-relay.kube-system.svc.cluster.local
+  port: 80
+
+kubearmor:
+  url: kubearmor.kube-system.svc.cluster.local
+  port: 32767
```
Comment (nyrahul) [inline knox-auto-policy-chart/configmapfiles/conf.yaml]: Can we also have namespace configurable for cilium and kubearmor?
```
--cilium-ns xyz
--kubearmor-ns xyz
```
default is `kube-system`

File: knox-auto-policy-chart/values.yaml [hunk 2]
```diff
@@ -95,75 +95,72 @@ tolerations: []
...
-      events:
-        buffer: 50
-
-  logging:
-    level: "INFO"
-    
-  cilium-hubble:
-    url: 10.4.41.240
-    port: 80
+# config:
```
Comment (nyrahul) [inline knox-auto-policy-chart/values.yaml]: can we delete these commented configs?

**Review by nyrahul — CHANGES_REQUESTED** [review]

File: knox-auto-policy-chart/configmapfiles/conf.yaml (hunk 1 above)
Comment (SujithKasireddy) [reply knox-auto-policy-chart/configmapfiles/conf.yaml]: okay

File: knox-auto-policy-chart/values.yaml (hunk 2 above)
Comment (SujithKasireddy) [reply knox-auto-policy-chart/values.yaml]: okay

Comment (SujithKasireddy) [reply knox-auto-policy-chart/values.yaml]: @nyrahul  I have made changes, please review it

**Review by nyrahul — APPROVED** [review]

**Pull request merged**